### DISCUSSION TITLE
学習コンテンツにVS Codeの設定のガイドを追加する

### DISCUSSION THREAD

**Discussion by tani-takanori** [ws-4020/mobile-app-crib-notes#309] [Ideas]
https://github.com/ws-4020/mobile-app-crib-notes/pull/305

保存したときに修正すると
 - code例をわざわざprettier準拠にしなくていい
 - diffも自由にかけるのでわかりやすくなる
 といった効果があると思います

ESLintのfixをかけた方が良いと思うんですけど、VS Codeだったらこんな感じで設定してfixしてくれませんかね？
```
{
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  }
}
```
参考: https://qiita.com/mysticatea/items/3f306470e8262e50bb70#editorcodeactionsonsave

**Comment by creature-water-valley** [comment]
以下のPRで対応済み
- https://github.com/ws-4020/mobile-app-crib-notes/pull/1285
- https://github.com/ws-4020/rn-spoiler/pull/151
